demisto_sdk/commands/common/tools.py

```diff
@@ -3724,3 +3727,30 @@ def parse_multiple_path_inputs(
         return result
 
     raise ValueError(f"Cannot parse paths from {input_path}")
+
+
+def md5_update_from_file(filename: Union[str, Path], hash):
+    assert Path(filename).is_file()
+    with open(str(filename), "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hash.update(chunk)
+    return hash
+
+
+def md5_file(filename: Union[str, Path]) -> str:
+    return str(md5_update_from_file(filename, hashlib.md5()).hexdigest())
+
+
+def md5_update_from_dir(directory: Union[str, Path], hash_):
+    assert Path(directory).is_dir()
+    for path in sorted(Path(directory).iterdir(), key=lambda p: str(p).lower()):
+        hash_.update(path.name.encode())
+        if path.is_file():
+            hash_ = md5_update_from_file(path, hash_)
+        elif path.is_dir():
+            hash_ = md5_update_from_dir(path, hash_)
+    return hash_
+
+
+def md5_dir(directory: Union[str, Path]) -> str:
+    return str(md5_update_from_dir(directory, hashlib.md5()).hexdigest())
```

Comment:
1. add unit tests please
2. is there any tool we can use instead of writing it ourselves?

Reply:
there are tools for it, but this is straightforward and I don't think we need to add a dependency just for it.
This is inspired from this thread: https://stackoverflow.com/questions/24937495/how-can-i-calculate-a-hash-for-a-filesystem-directory-using-python

Reply:
added UT